package.json

```diff
@@ -27,7 +28,9 @@
     "sinon": "^1.17.5",
     "valid-url": "1.0.9"
   },
+  "types:":"./types/fh-sync.d.ts",
   "scripts": {
+    "docs": "./node_modules/.bin/jsdoc -d apidoc ./lib",
```

Comment:
Good catch.. I was using jsdocs to generate documentation to validate typings and forgot to remove it.